universal-login-commons/lib/core/utils/hexStrings.ts

```diff
@@ -1,5 +1,9 @@
 import {ensure} from './errors';
 
+export const isProperHexString = (hexString: string) => {
+  return hexString.startsWith('0x') && hexString.length % 2 === 0;
```

Comment:
Doesn't check the conent of string, simpler:
  return !!address.match(/^0x[0-9A-Fa-f]+$/);